From: Markus Koschany <apo@debian.org>
Date: Fri, 11 May 2018 20:55:47 +0200
Subject: java10

Ignore the error at build time. Hopefully there are no runtime issues..

Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=893259
Forwarded: no
---
 build/build.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/build/build.xml b/build/build.xml
index 4ebfb29..734bef1 100644
--- a/build/build.xml
+++ b/build/build.xml
@@ -205,7 +205,7 @@ file has been tested with ant version 1.6.1.
         <path refid="support-classpath"/>
       </classpath>
     </javac>
-    <java classname="org.jibx.binding.Compile" fork="yes" failonerror="true">
+    <java classname="org.jibx.binding.Compile" fork="yes" failonerror="false">
       <!-- This command line argument should only be used when binding JiBX.
         It skips the step of validating the binding using the binding definition
         model (because the model itself uses JiBX binding, which can create
@@ -242,7 +242,7 @@ file has been tested with ant version 1.6.1.
         <fileset dir="/usr/share/java" includes="log4j-1.2.jar" />
       </classpath>
     </javac>
-    <java classname="org.jibx.binding.Compile" fork="yes" failonerror="true">
+    <java classname="org.jibx.binding.Compile" fork="yes" failonerror="false">
       <classpath>
         <pathelement path="${schemadest}"/>
         <fileset dir="${lib}" includes="*.jar" excludes="asm*.jar,org.eclipse.*.jar,qdox*.jar,${schemaname}.jar,${toolsname}.jar"/>
@@ -270,7 +270,7 @@ file has been tested with ant version 1.6.1.
         <fileset dir="${lib}" includes="*.jar" excludes="asm*.jar,${toolsname}.jar"/>
       </classpath>
     </javac>
-    <java classname="org.jibx.binding.Compile" fork="yes" failonerror="true">
+    <java classname="org.jibx.binding.Compile" fork="yes" failonerror="false">
       <classpath>
         <pathelement path="${toolsdest}"/>
         <fileset dir="${lib}" includes="*.jar" excludes="asm*.jar,${toolsname}.jar"/>
